app/Http/Controllers/EngagementController.php

```diff
@@ -154,21 +149,22 @@ public function showCriteriaSelection(Engagement $engagement): View
             'project' => $engagement->project,
             'engagement' => $engagement,
             'regions' => Options::forEnum(ProvinceOrTerritory::class)->toArray(),
-            'crossDisability' => DisabilityType::where('name->en', 'Cross-disability')->first(),
-            'disabilityTypes' => Options::forModels(DisabilityType::query()->where([
-                ['name->en', '!=', 'Cross-disability'],
+            'crossDisability' => Identity::where('name->en', 'Cross-disability and Deaf')->first(),
+            'disabilityTypes' => Options::forModels(Identity::query()->where([
+                ['cluster', IdentityCluster::DisabilityAndDeaf],
+                ['name->en', '!=', 'Cross-disability and Deaf'],
                 ['name->en', '!=', 'Temporary disabilities'],
             ]))->toArray(),
-            'ageBrackets' => Options::forModels(AgeBracket::class)->toArray(),
-            'areaTypes' => Options::forModels(AreaType::class)->toArray(),
-            'indigenousIdentities' => Options::forModels(IndigenousIdentity::class)->toArray(),
-            'ethnoracialIdentities' => Options::forModels(EthnoracialIdentity::query()->where('name->en', '!=', 'White'))->toArray(),
-            'women' => GenderIdentity::where('name_plural->en', 'Women')->first(),
-            'nb' => GenderIdentity::where('name_plural->en', 'Non-binary people')->first(),
-            'gnc' => GenderIdentity::where('name_plural->en', 'Gender non-conforming people')->first(),
-            'fluid' => GenderIdentity::where('name_plural->en', 'Gender fluid people')->first(),
-            'transPeople' => Constituency::where('name_plural->en', 'Trans people')->first(),
-            'twoslgbtqiaplusPeople' => Constituency::where('name_plural->en', '2SLGBTQIA+ people')->first(),
+            'ageBrackets' => Options::forModels(Identity::query()->where('cluster', IdentityCluster::Age))->toArray(),
+            'areaTypes' => Options::forModels(Identity::query()->where('cluster', IdentityCluster::Area))->toArray(),
+            'indigenousIdentities' => Options::forModels(Identity::query()->where('cluster', IdentityCluster::Indigenous))->toArray(),
+            'ethnoracialIdentities' => Options::forModels(Identity::query()->where('cluster', IdentityCluster::Ethnoracial))->reject(fn (Identity $identity) => $identity->name === __('White'))->toArray(),
+            'women' => Identity::where('name->en', 'Women')->first(),
+            'nb' => Identity::where('name->en', 'Non-binary people')->first(),
+            'gnc' => Identity::where('name->en', 'Gender non-conforming people')->first(),
+            'fluid' => Identity::where('name->en', 'Gender fluid people')->first(),
+            'transPeople' => Identity::where('name->en', 'Trans people')->first(),
+            'twoslgbtqiaplusPeople' => Identity::where('name->en', '2SLGBTQIA+ people')->first(),
```

Comment:
I'm not sure this works if we are expecting that new identities can be added via filament. I think you may need to pass back the sets of identities or maybe all the identities grouped by cluster.

Reply:
This is one of the challenges we need to address— to discuss tomorrow.